sdk/tables/azure-data-tables/src/test/resources/session-records/AzureTableImplTest.createTableDuplicateNameImpl.json

```diff
@@ -1,94 +1,161 @@
 {
-  "networkCallRecords" : [ {
-    "Method" : "POST",
-    "Uri" : "https://REDACTED.table.core.windows.net/Tables",
-    "Headers" : {
-      "x-ms-version" : "2019-02-02",
-      "x-ms-client-request-id" : "87789852-9f83-4cc3-a0d2-a8ad2ede0312",
-      "Content-Type" : "application/json;odata=nometadata"
+  "Entries": [
+    {
+      "RequestUri": "https://REDACTED/Tables",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "Content-Length": "30",
+        "Content-Type": "application/json;odata=nometadata",
+        "DataServiceVersion": "3.0",
+        "Date": "Tue, 11 Apr 2023 21:19:02 GMT",
+        "Prefer": "return-no-content",
+        "User-Agent": "ReactorNetty/dev",
+        "x-ms-client-request-id": "46298e9a-3107-4a6f-8358-b5ea2353dd2a",
+        "x-ms-version": "2020-12-06"
+      },
+      "RequestBody": {
+        "TableName": "test8031195858"
+      },
+      "StatusCode": 204,
+      "ResponseHeaders": {
+        "Cache-Control": "no-cache",
+        "Content-Length": "0",
+        "DataServiceId": "https://tb914c4cd1614894eprim.table.core.windows.net/Tables(\u0027test8031195858\u0027)",
+        "Date": "Tue, 11 Apr 2023 21:19:01 GMT",
+        "Location": "https://tb914c4cd1614894eprim.table.core.windows.net/Tables(\u0027test8031195858\u0027)",
```

Comment:
We should sanitize this.
    private static final String URL_REGEX = "(?<=http://|https://)([^/?]+)";
new TestProxySanitizer("Location", URL_REGEX, "REDACTED", HEADER)